chunky/src/java/se/llbit/chunky/chunk/BlockPalette.java

```diff
@@ -440,23 +443,26 @@ public static Map<String, Consumer<Block>> getDefaultMaterialProperties() {
       block.emittance = 0.6f;
     });
     materialProperties.put("minecraft:soul_fire_torch", block -> { // MC 20w06a-20w16a
-      block.emittance = 35.0f;
+      block.emittance = 3.5f;
     });
     materialProperties.put("minecraft:soul_torch", block -> { // MC >= 20w17a
-      block.emittance = 35.0f;
+      block.emittance = 3.5f;
     });
     materialProperties.put("minecraft:soul_fire_wall_torch", block -> { // MC 20w06a-20w16a
-      block.emittance = 35.0f;
+      block.emittance = 3.5f;
     });
     materialProperties.put("minecraft:soul_wall_torch", block -> { // MC >= 20w17a
-      block.emittance = 35.0f;
+      block.emittance = 3.5f;
     });
     materialProperties.put("minecraft:soul_fire", block -> {
       block.emittance = 0.6f;
     });
     materialProperties.put("minecraft:crying_obsidian", block -> {
       block.emittance = 0.6f;
     });
+    materialProperties.put("minecraft:enchanting_table", block -> {
+      block.emittance = 0.6f;
```

Comment:
Crying obsidian has a light level of _10_ and emittance _0.6_. Enchanting table has a light level of 7, which would be 0.46 (round as you see fit) in Chunky.

Reply:
Enchanting table emittance now set to 0.5